README.md

```diff
@@ -36,7 +36,7 @@
 Full-stack web application framework that uses Python and MariaDB on the server side and a tightly integrated client side library. Built for [ERPNext](https://erpnext.com).
 
 <div align="center" style="max-height: 40px;">
-	<a href="https://frappecloud.com/frappe/signup">
```

Comment:
This link works fine rn. This is SaaS signup link which skips whole bunch of steps and just gives a trial site. Let't not change it to FC signup.

Reply:
@ankush huh? It’s a 404 for me.